scopus-generate/src/main/resources/xocs-ani515.xjb

```diff
@@ -0,0 +1,24 @@
+<bindings xmlns="https://jakarta.ee/xml/ns/jaxb"
```

Comment:
Remember to remove create-publication-from-scopus/src/main/resources/xocs-ani515.xjb